warema_wms/wms_controller.py

```diff
@@ -18,16 +18,17 @@
 RX_SHADE_STATE = '0431'
 
 TX_MOVE_SHADE = '0821'
-SHADE_POSITION = '03{}ffffff'
+SHADE_POSITION = '03{}dfffff'   # df = trial to move raffstores in tilt closed pos., working in console but not in HA
+TILT_POSITION = '03ff{}ffff'
+STOP_SHADE = '01ffffffff'       # at least at my setup it works (instead of STOP_SHADE = '0001ffffff')
 
 logger = logging.getLogger('warema_wms')
-
+logging.debug
```

Comment:
```suggestion
```

This should probably not be here.

Reply:
yep, dont have any idea what my intention was with this, sorry